app/presenters/image_upload_response_presenter.rb

```diff
@@ -53,6 +53,8 @@ def as_json(*)
       json[:ocr_pii] = ocr_pii
       json[:result_failed] = doc_auth_result_failed?
       json[:doc_type_supported] = doc_type_supported?
+      json[:selfie_live] = selfie_live? if show_selfie_failures
+      json[:selfie_quality_good] = selfie_quality_good? if show_selfie_failures
```

Comment:
I feel like only including these in the JSON when the selfie is required makes sense? But that might be bad practice?

Reply:
@charleyf this is the then rendered as http response to FE, if you need it, you need include it?